src/core/utils.h

```diff
@@ -81,7 +81,7 @@ inline void getPolicyInTensor(const State& state,
   auto maskaccessor = mask.accessor<float, 3>();
 
   const auto& legalAction = state.GetLegalActions();
-  for (mcts::Action actionIdx = 0; actionIdx != pi.size(); ++actionIdx) {
+  for (mcts::Action actionIdx = 0; actionIdx != static_cast<mcts::Action>(pi.size()); ++actionIdx) {
```

Comment:
shouldn't this be `<` as opposed to `!=` ?

Reply:
I think `!=` is fine, it only increases by 1, there are no other threads. But fair enough :)